config/environments/production.rb

```diff
@@ -80,4 +80,17 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # mandrill config stuff, depends on config/initializers/constants.rb
+  config.action_mailer.smtp_settings = {
```

Comment:
this is where i usually put these, but yeah, since it's using env variables for every environment, it could work in an initializer.